### DISCUSSION TITLE
New Diagram - Support for Open Group ArchiMate

### DISCUSSION THREAD

**Discussion by demit733** [mermaid-js/mermaid#4007] [Ideas]
I would like to request Mermaid support the OpenGroup ArchiMate diagrams. 

ArchiMate is becoming the stand for Enterprise Architecture. ArchiMate is made up of server sets of diagrams that supports the full stack of software development, all the way from conceptual (Motivation) phase through deployment(Implementation). See below attachment: 
![ArchiMate 3 0 Notation Overview](https://user-images.githubusercontent.com/112209110/213034402-e78481a5-b728-4edb-a93b-92d12f058ad4.png)

**Comment by Hubbitus** [comment]
Very good idea.

**Comment by glewellyn** [comment]
This would be really useful!
While the Archimate XML format is very good for transferring diagrams from one modeling tool to another, it is not very suitable for automated creation of new diagrams.

**Comment by neptuniux** [comment]
What is the status of this feature? Is there a workaround if it is not implemented?

**Comment by michalradacz** [comment]
I start creating SLL for ArchiMate, next step will be generating scripts for ANY drawing language, especially for Mermaid.
Look after some days at https://github.com/michalradacz/archimate4drawings

**Comment by FantasyNitroGEN** [comment]
Up!

**Comment by Polyhistor** [comment]
This would be great!

**Comment by Oliveiras** [comment]
Any news on Archimate support? This would make Mermaid "the PlantUML killer"  for me.

**Comment by BettaiebMohamed** [comment]
This would be a great addition given how common ArchiMate is in enterprise architecture.

Maybe a good starting point is supporting just the core layers (Business, Application, Technology) and expanding later. Another option could be a mapping/DSL approach that translates ArchiMate concepts into existing Mermaid diagrams.

Curious if this fits Mermaid’s scope or would be better as a plugin first?

**Comment by koayact** [comment]
For now, refer to <a href="https://mermaid.ai/open-source/syntax/architecture.html">Mermaid - Architecture <i>(beta)</i> diagram-as-code</a> which I'm exploring through the work at https://github.com/koayact/D3js/wiki/BIAN-Service-Landscape for illustration only.
